doc/flux_to_redux.md

```diff
@@ -0,0 +1,579 @@
+
+    Despite there being four stores, all of them have the same job: handle a single random color property. Therefore, they are all nearly identical objects. Naturally in the real world they would likely be drastically different, but for simplicity sake and keeping the focus on the big picture they are all left the same.
+- __The Store Objects:__ `bigStore` and `littleStore`
+- __The Actions Class__, `RandomColorActions`
+- __The Actions Object__, `randomColorActions`
+- __The UI:__ The only complex component is a `BigBlock` component that receives `randomColorActions`, `bigStore`, and `littleStore` as props. Updates are triggered by using the action prop. Naturally now state is accessed via:
+    - props.bigStore.state
+    - props.bigStore.midLevelStore.state
+    - props.bigStore.midLevelStore.lowLevelStore.state
+    - props.littlebig_Store.state
+
+    Our `BigBlock` component renders three `SmallBlock` components. Each one connects to a different store's state, with the remaining background color being tied to `BigBlock` itself. 
+    
+    `BigBlock` also has four buttons - one connected to each action that changes the background color property of one of the stores (and thereby updating the component background color).
+
+    A Flux specific part of the `BigBlock` component is that we need to set the `redrawOne` to listen to the story at every level.
+
+```dart
+// A nested store
+class RandomColorStore extends flux.Store {
+    RandomColorActions _actions;
+    // MidLevelStore looks nearly identical to `RandomColorStore`, but has a different
+    // store object (`LowLevelStore`) nested within it.
+    MidLevelStore midLevelStore = MidLevelStore(randomColorActions);
+
+    /// Public data
+    String _backgroundColor = 'gray';
+    String get backgroundColor => _backgroundColor;
+
+    RandomColorStore(this._actions) {
+        triggerOnActionV2(_actions.changeMainBackgroundColor, _changeBackgroundColor);
+    }
+
+    _changeBackgroundColor(String _) {
+        _backgroundColor = '#' + (Random().nextDouble() * 16777215).floor().toRadixString(16);
+    }
+}
+
+// A seperate store, making the total store count be equal to 4
+class AnotherColorStore extends flux.Store {
+    RandomColorActions _actions;
+
+    /// Public data
+    String _backgroundColor = 'Blue';
+    String get backgroundColor => _backgroundColor;
+
+    AnotherColorStore(this._actions) {
+        triggerOnActionV2(_actions.changeBlockThreeBackgroundColor, _changeBackgroundColor);
+    }
+
+    _changeBackgroundColor(String _) {
+        _backgroundColor = '#' + (Random().nextDouble() * 16777215).floor().toRadixString(16);
+    }
+}
+```
+
+```dart
+// A shared action class for all the stores, with an action to update each background color
```

Comment:
I think we should show the necessary imports for clarity as to what the `flux` namespace represents.

```suggestion
import 'package:w_flux/w_flux.dart' as flux;
    
// A shared action class for all the stores, with an action to update each background color
```